### PR TITLE
Cosmetic RPM specfile fix

### PR DESCRIPTION
**Pull request by onuralpszr**

### CODE REVIEW

File: nvautoinstall.spec [hunk 1]
```diff
@@ -7,13 +7,17 @@ Summary: NVIDIA Auto Installer for Fedora
 
 License: GPLv3
 Url: https://github.com/t0xic0der/%{srcname}
-Source0: https://github.com/t0xic0der/%{srcname}/releases/download/%{version}/%{name}-%{version}.tar.gz
+Source0: https://github.com/t0xic0der/%{srcname}/releases/download/v%{version}/%{name}-%{version}.tar.gz
```
Comment (gridhead) [inline nvautoinstall.spec:10]: Using the following replacements,

1. `srcname` - `nvidia-auto-installer-for-fedora`
2. `name` - `nvautoinstall`
3. `version` - `0.3.8`

`https://github.com/t0xic0der/nvidia-auto-installer-for-fedora/releases/download/0.3.8/nvautoinstall-0.3.8.tar.gz` throws a `404 Not Found` while `https://github.com/t0xic0der/nvidia-auto-installer-for-fedora/releases/download/v0.3.8/nvautoinstall-0.3.8.tar.gz` gets the tarball just fine.

Thanks for the change.

File: nvautoinstall.spec [hunk 2]
```diff
@@ -7,13 +7,17 @@ Summary: NVIDIA Auto Installer for Fedora
...
 
 BuildArch: noarch
 
 BuildRequires: python3-devel
 BuildRequires: python3-setuptools
 
+Requires: python3-click
+Requires: python3-distro
```
Comment (gridhead) [inline nvautoinstall.spec]: This is listed here

https://github.com/t0xic0der/nvidia-auto-installer-for-fedora/blob/45a703fc8741f05c1daad3d7c513d5c5bd334a02/setup.py#L35-L38

Any specific reason as to why we would want to have it here too?

File: nvautoinstall.spec [hunk 3]
```diff
@@ -37,7 +41,7 @@ A CLI tool which lets you install proprietary NVIDIA drivers and much more
 #-- CHANGELOG -----------------------------------------------------------------#
 %changelog
 
-* Wed Aug 02 2021 Akashdeep Dhar <t0xic0der@fedoraproject.org>
+* Mon Aug 02 2021 Akashdeep Dhar <t0xic0der@fedoraproject.org>
```
Comment (gridhead) [inline nvautoinstall.spec:40]: Thanks for the change. :+1:

**Review by gridhead — COMMENTED** [review]
This PR is good for merge. Please answer one question in the review before we go ahead.

File: nvautoinstall.spec (hunk 2 above)
Comment (onuralpszr) [reply nvautoinstall.spec]: I missed it, it is reverted. thank you for pointing out.

File: nvautoinstall.spec (hunk 1 above)
Comment (onuralpszr) [reply nvautoinstall.spec:10]: :+1:

File: nvautoinstall.spec (hunk 3 above)
Comment (onuralpszr) [reply nvautoinstall.spec:40]: :+1:

**Review by gridhead — APPROVED** [review]
LGTM. Thank you for the pull request.

**Pull request merged**